drivers/soundwire/stream.c

```diff
@@ -281,6 +281,9 @@ static int sdw_program_port_params(struct sdw_master_runtime *m_rt)
 	/* Program transport & port parameters for Slave(s) */
 	list_for_each_entry(s_rt, &m_rt->slave_rt_list, m_rt_node) {
 		list_for_each_entry(p_rt, &s_rt->port_list, port_node) {
+			ret = sdw_slave_set_frequency(s_rt->slave);
```

Comment:
in my quick reading of the function, it will always program registers, even if they don't change, is that OK?

Reply:
> in my quick reading of the function, it will always program registers, even if they don't change, is that OK?

Yes, it should be ok